Tools/BatchScripts/batch_cori_gpu.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -l
+
+# Copyright 2021 Axel Huebl
+# This file is part of WarpX.
+# License: BSD-3-Clause-LBNL
+#
+# Ref:
+# - https://docs-dev.nersc.gov/cgpu/hardware/
+# - https://docs-dev.nersc.gov/cgpu/access/
+
+# Just increase this number of you need more nodes.
+#SBATCH -N 2
+#SBATCH -t 03:00:00
+#SBATCH -J <job name>
+#SBATCH -A <allocation ID>
```

Comment:
Since this is the only allocation that supports GPUs atm, let's just write it explicitly:
```suggestion
#SBATCH -A m1759
```